_targets/build.project.ia32-generic

```diff
@@ -31,11 +34,52 @@ export PORTS_JANSSON=y
 export PORTS_CURL=y
 
 
+#
+# Platform dependent parameters
+#
+SIZE_PAGE=$((0x1000))
+PAGE_MASK=$((0xfffff000))
```

Comment:
We can define `PAGE_MASK` using `SIZE_PAGE`
`~(SIZE_PAGE - 1)`

Reply:
In bash `PAGE_MASK=$((~(SIZE_PAGE - 1)` gets interpreted as signed integer (`echo $PAGE_MASK` prints `-4096`).
Should I define it as `PAGE_MASK=$((~(SIZE_PAGE - 1) & 0xffffffff))` to 'cast' it to unsigned integer?

Reply:
I used it like that: https://github.com/phoenix-rtos/phoenix-rtos-build/blob/c6491a4144ee51d0760afec156d8669e5e425fe8/scripts/mkimg-stm32.sh#L51

It seem to work just fine in my case

Reply:
It's fine as long as we're doing only bitwise operations with it (which is true in our case). I've added the proposed change.